### PR TITLE
doc: add package.conf example, closes HYB-559

### PR DESCRIPTION
**Pull request by Obero**

### CODE REVIEW

**Comment by Obero** [comment]
Draft until feature release

File: .gatling/example.package.conf [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          "com.example.prop.1" = "override value from system prop 1"
+        }
+        environmentVariables {
+          MY_SIMULATION_ENV_VAR_1 = "override value from environment 1"
+        }
+      }
+      timeWindow {
+        rampDown = 20
+      }
+    }
```
Comment (tpetillot) [inline .gatling/example.package.conf:62]: I would prefer a working as it is configuration, so without this, or with SimulationB (or such) existing 😅

Comment (bastien-gatling) [reply .gatling/example.package.conf:62]: Actually, I would only put in the example the existing simulation in the project.

File: .gatling/example.package.conf [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
+
```
Comment (Obero) [inline .gatling/example.package.conf:2]: https://github.com/gatling/gatling-gradle-plugin-demo-java/pull/17#discussion_r1560533903

> I would add a new comment line here, like : `All configurations are optional`

File: .gatling/example.package.conf [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          size: 2,
+          weight: 70
+        }
+      ]
+      parameters {
+        ignoreDefaults = false
+        systemProperties {
+          "com.example.prop.1" = "default value from system prop 1"
+          "com.example.prop.2" = "default value from system prop 2"
+        }
```
Comment (Isammoc) [inline .gatling/example.package.conf:28]:
```suggestion
        systemProperties {
          # key should be surrounded by quotes
          "com.example.prop.1" = "default value from system prop 1"
          "com.example.prop.2" = "default value from system prop 2"
        }
```

Add a comment to explain why the key should be surrounded by quotes?
(didn't find a good explanation in english)

File: .gatling/example.package.conf [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        environmentVariables {
+          MY_SIMULATION_ENV_VAR_1 = "default value from environment 1"
+          MY_SIMULATION_ENV_VAR_2 = "default value from environment 2"
+        }
+      }
+      timeWindow {
+        rampUp = 10
+        rampDown = 10
+      }
```
Comment (Isammoc) [inline .gatling/example.package.conf:37]: A comment for the unit?
```suggestion
      timeWindow {
        rampUp = 10 # in seconds
        rampDown = 10 # in seconds
      }
```

File: .gatling/example.package.conf [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    }
+  }
+  simulations = [
+    {
+      # Consistent deployment using an existing simulation ID
+      # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+      # id = "00000000-0000-0000-0000-000000000001"
+      classname = "computerdatabase.ComputerDatabaseSimulation"
```
Comment (Isammoc) [inline .gatling/example.package.conf:45]:
```suggestion
      # FQCN Fully Qualified Class Name
      classname = "computerdatabase.ComputerDatabaseSimulation"
```

File: .gatling/package.conf [hunk 6]
```diff
@@ -0,0 +1,2 @@
+# See `example.package.conf` for a full deployment configuration file example
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```
Comment (Isammoc) [inline .gatling/package.conf:2]: No space before semi colon:

```suggestion
# Learn more about Configuration as Code: https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```

File: .gatling/example.package.conf [hunk 7]
```diff
@@ -0,0 +1,64 @@
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```
Comment (Isammoc) [inline .gatling/example.package.conf:1]: No space before semi-colon:

```suggestion
# Learn more about Configuration as Code: https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```

Comment (Isammoc) [inline .gatling/example.package.conf:1]: I guess the url is not yet valid because based on PR?

Comment (Obero) [reply .gatling/example.package.conf:1]: Yep, it will be valid after documentation is published

File: .gatling/package.conf (hunk 6 above)
Comment (Isammoc) [reply .gatling/package.conf:2]: Still here

File: .gatling/example.package.conf [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+        {
+          name: "AP Pacific - Mumbai",
+          size: 2,
+          weight: 70
+        }
+      ]
+      parameters {
+        ignoreDefaults = false
+        systemProperties {
+          # System properties names should be surroinded by quotes
```
Comment (Isammoc) [inline .gatling/example.package.conf:28]:
```suggestion
          # System properties names should be surrounded by quotes
```

File: .gatling/example.package.conf [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+          # Otherwise, it would be interpreted as HOCON keys
```
Comment (Isammoc) [inline .gatling/example.package.conf:29]: nice :+1:

File: .gatling/example.package.conf (hunk 8 above)
Comment (Obero) [reply .gatling/example.package.conf:28]: I'm playing with the new PR comments integration in IDEA, it seems I failed :smile:

File: .gatling/package.conf (hunk 6 above)
Comment (Obero) [reply .gatling/package.conf:2]: I'm playing with the new PR comments integration in IDEA, it seems I failed 😄

**Review by Isammoc — APPROVED** [review]

**Review by bastien-gatling — APPROVED** [review]

**Review by tpetillot — APPROVED** [review]

File: .gatling/example.package.conf [hunk 10]
```diff
@@ -0,0 +1,54 @@
+# Learn more about Configuration as Code: https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
+# This file is full deployment configuration file example.
+# Please note all these configuration fields, or even this file, are optional.
+
+gatling.enterprise.package {
+  # Consistent deployment using an existing package ID
+  # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+  # id = "00000000-0000-0000-0000-000000000000"
+  name = "My package name"
+  team = "My team name" # or ID with team = "00000000-0000-0000-0000-000000000000"
```
Comment (tpetillot) [inline .gatling/example.package.conf:10]:
```suggestion
  team = "Default team" # or ID with team = "00000000-0000-0000-0000-000000000000"
```

File: .gatling/example.package.conf [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        rampDown = 10 # in seconds
+      }
+    }
+  }
+  simulations = [
+    {
+      # Consistent deployment using an existing simulation ID
+      # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+      # id = "00000000-0000-0000-0000-000000000001"
+
```
Comment (tpetillot) [inline .gatling/example.package.conf:49]:
```suggestion
  name = "My simulation name"
```

**Review by tpetillot — CHANGES_REQUESTED** [review]

**Review by tpetillot — APPROVED** [review]

**Pull request merged**